src/sentry/api/bases/organization_events.py

```diff
@@ -202,6 +204,11 @@ def handle_data(self, request, organization, project_ids, results):
                 if has_issues and "issue.id" in result:
                     result["issue"] = issues.get(result["issue.id"], "unknown")
 
+        for result in results:
+            for key in result:
+                if isinstance(result[key], numbers.Number) and not math.isfinite(result[key]):
+                    result[key] = None
```

Comment:
If we've also run into this with query subscriptions should this be done lower down closer to where results are parsed from snuba?

Reply:
Here? https://github.com/getsentry/sentry/blob/c11e169371a618257cb18e13b4499f73ca8081be/src/sentry/utils/snuba.py#L719-L721

Reply:
Yeah that could work.